src/symbol.rs

```diff
@@ -0,0 +1,49 @@
+use super::{Parse, Token, TokenKind};
+
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub struct Symbol;
+
+impl Parse for Symbol {
+    fn parse(input: &str) -> Option<Token> {
+        let symbol = match input.chars().next()? {
+            '{' => TokenKind::OpenBrace,
+            '}' => TokenKind::CloseBrace,
+            '[' => TokenKind::OpenBracket,
+            ']' => TokenKind::OpenBracket,
```

Comment:
```suggestion
            ']' => TokenKind::CloseBracket,
```

Missed this - and I copy pasted the code to make the test :smile:.